index.html

```diff
@@ -27,19 +27,19 @@ <h1>My Projects</h1>
       <div id="main-con">
```

Comment:
Kindly make use of the section tag to group standalone related content. Your navbar, content, footer,  should be enclosed in section tags.